nemo_text_processing/inverse_text_normalization/he/data/numbers/decimal_fractions.tsv

```diff
@@ -0,0 +1,9 @@
+חצי	5
+רבע	25
+שלושת רבעי	75
+שליש	333
+שני שליש	67
```

Comment:
This is "two thirds" (.67) in Hebrew, this is what it would be translated to as a decimal number

Reply:
As in, a native speaker would see 67 and it's valid to say "two-thirds"? Or you're just rounding up the two thirds?

Reply:
When you talk about "two thirds" and you expect to see the number with a decimal point, than it is either `.067` or `.66`.
Since it is not converted to fractions (`2/3`), I don't think it really matters which of the options we will choose. It could also be `.667` if it is better.

Reply:
Got it. So in this case we shouldn't include two thirds for decimals. Part of the rationale for rule-based normalization is that they should be brutally reliable for numerical tasks. Having a conversion that includes rounding would go against that.

Reply:
Thanks, sorry for being a stickler but it's a downstream restriction that we have to abide by.